resource_pool/starter.py

```diff
@@ -0,0 +1,66 @@
+import asyncio
+from typing import Any
+
+from temporalio.client import Client, WorkflowFailureError, WorkflowHandle
+from temporalio.common import WorkflowIDConflictPolicy
+
+from resource_pool.resource_pool_workflow import (
+    ResourcePoolWorkflow,
+    ResourcePoolWorkflowInput,
+)
+from resource_pool.resource_user_workflow import (
+    ResourceUserWorkflow,
+    ResourceUserWorkflowInput,
+)
+from resource_pool.shared import RESOURCE_POOL_WORKFLOW_ID
+
+
+async def main() -> None:
+    # Connect client
+    client = await Client.connect("localhost:7233")
+
+    # Initialize the resource pool
+    resource_pool_handle = await client.start_workflow(
+        workflow=ResourcePoolWorkflow.run,
+        arg=ResourcePoolWorkflowInput(
+            resources={"resource_a": None, "resource_b": None},
+            waiters=[],
+        ),
+        id=RESOURCE_POOL_WORKFLOW_ID,
+        task_queue="default",
```

Comment:
```suggestion
        task_queue="resource_pool-task-queue",
```

Here and everywhere just to avoid clash with other samples